web/geo/generate-aggregates.js

```diff
@@ -35,8 +35,12 @@ const generateAggregates = (geojson, zones) => {
     const [multiZoneCountry] = unCombinedZones.filter((feature) => feature.properties.zoneName === country[0]);
     for (let i = 0; i < country.length; i++) {
       const [zoneToAdd] = unCombinedZones.filter((feature) => feature.properties.zoneName === country[i]);
-
-      combinedCountry.geometry = union(combinedCountry.geometry, zoneToAdd.geometry).geometry;
+      if (zoneToAdd) {
+        combinedCountry.geometry = union(combinedCountry.geometry, zoneToAdd.geometry).geometry;
+      } else {
+        process.stdout.write(`ERROR: Could not find a feature for ${country[i]}`);
```

Comment:
@madsnedergaard we could probably use this in all the "script" files to get around the ESLint errors for `console`. What do you think?

Reply:
Hmm, it seems like previously `console.log` was just `process.stdout.write()` with `/n` appended, but when I look at the source code now it seems like more is happening: https://github.com/nodejs/node/blob/main/lib/internal/console/constructor.js

It is annoying with the errors though, maybe we can allow logging for any file in a future `/scripts` folder instead? :)

Reply:
Part of the current issue is they are a bit scatered right now. However if we move them to one place (or switch to a strict naming scheeme like `<filename>.script.js`) it should be pretty easy to add a ESLint overwrite.

If we decide to do that I'll change it to a simple Console statment and create a seperate PR for the other changes.

Reply:
Hmm yeah, but I guess it also makes sense to have this particular script in `/geo` - so maybe the naming convention is the best option? I'd also be fine with switching to process.stdout/stderr throughout though :)

Reply:
Yeah I think the best would be to create a naming convention as it would allow us to add rules that only target the local scripts (or exclude them like in this case).

We'll just have to agree on a naming convention that makes sense.

Reply:
I think console.error and console.warn is still available no?

Reply:
Looks like you are right!
```js
 'no-console': ['error', { allow: ['error', 'warn'] }],
 ```
 If we add info (`console.info()`) to that we could probably use it without the disable line comments in production as well (console.info() is lower priority than log, warn or error). Allthough I guess we should aim to remove all log messages from the frontend and report criticals errors to sentry instead, but that's another discussion all together.

Reply:
We could consider adding info there. Right now I like that it's a bit strict because it forces you to be intentional but it makes less sense in scripts like these

Reply:
So a naming convention and ESLint overrides might still be best?
I.e. allow all console statments in "script" files and only allow warn and error in the frontend?

Reply:
We can always add an override and naming convention later if we have any other conditional rules so if @Kongkille don't have any objections I'll create a PR for `.info` and update the console statments tomorrow. 🙂